modules/shared/client/services/logger.service.ts

```diff
@@ -23,30 +23,36 @@ export class LoggerService {
     this.http.post(apiEndpoint, { level: level, args: args })
       .map((res) => {
         return res.json();
+      }).subscribe((data) => {
+        console.log(data);
```

Comment:
yeah....  I almost dinged on this one... but I don't know of another way to log a log statment o_O